csharp/src/Google.Protobuf/MessageParser.cs

```diff
@@ -161,6 +164,33 @@ public IMessage ParseJson(string json)
             return message;
         }
 
+#if GOOGLE_PROTOBUF_SUPPORT_SPAN
+        /// <summary>
+        /// Parses a message from the given sequence.
+        /// </summary>
+        /// <param name="data">The data to parse.</param>
+        /// <returns>The parsed message.</returns>
+        public IMessage ParseFrom(ReadOnlySequence<byte> data)
+        {
+            IMessage message = factory();
+            message.MergeFrom(data, DiscardUnknownFields, Extensions);
+            return message;
+        }
+
+        /// <summary>
+        /// Parses a message from the given coded input stream.
+        /// </summary>
+        /// <param name="input">The stream to parse.</param>
+        /// <returns>The parsed message.</returns>
+        public IMessage ParseFrom(ref CodedInputReader input)
+        {
+            IMessage message = factory();
+            MergeFrom(message, ref input);
+            CheckMergedRequiredFields(message);
```

Comment:
I think this line was removed in https://github.com/protocolbuffers/protobuf/pull/6674/files#diff-1518fe4210577434130a7ae84f789c4eL75  (please adjust).